database-commons/src/main/java/io/cdap/plugin/db/sink/ETLDBOutputFormat.java

```diff
@@ -67,14 +70,27 @@ public RecordWriter<K, V> getRecordWriter(TaskAttemptContext context) throws IOE
     String tableName = dbConf.getOutputTableName();
     String[] fieldNames = dbConf.getOutputFieldNames();
     final int batchSize = conf.getInt(COMMIT_BATCH_SIZE, DEFAULT_COMMIT_BATCH_SIZE);
+    final String operationName = conf.get(OPERATION_NAME);
+    String relationTableKey = null;
+    String[] listKeys = new String[0];
```

Comment:
initialize it with nulll and null checks should be there in further methods where we are passing it as argument

Reply:
done